src/helpers/template-helper.js

```diff
@@ -0,0 +1,35 @@
+/*
+ *  *******************************************************************************
+ *  * Copyright (c) 2020 Edgeworx, Inc.
+ *  *
+ *  * This program and the accompanying materials are made available under the
+ *  * terms of the Eclipse Public License v. 2.0 which is available at
+ *  * http://www.eclipse.org/legal/epl-2.0
+ *  *
+ *  * SPDX-License-Identifier: EPL-2.0
+ *  *******************************************************************************
+ *
+ */
+// ninja2 like template engine
+var { Liquid } = require('liquidjs')
+let templateEngine = new Liquid()

```

Comment:
Why use `let` instead of `const`, the value of `templateEngine` never gets reassigned later on

Reply:
OK. I forget this two, npm run standard does not shown me the problem.